cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -297,19 +319,28 @@ pub mod pallet {
 	/// case of the need to send a high-priority signal message this block.
 	/// The bool is true if there is a signal message waiting to be sent.
 	#[pallet::storage]
-	pub(super) type OutboundXcmpStatus<T: Config> =
-		StorageValue<_, Vec<OutboundChannelDetails>, ValueQuery>;
+	pub(super) type OutboundXcmpStatus<T: Config> = StorageValue<
+		_,
+		BoundedVec<OutboundChannelDetails, T::MaxActiveOutboundChannels>,
```

Comment:
`Vec -> BoundedVec` the migration checks that `MaxActiveOutboundChannels` is above the current length of this.  
If governance decides to open more channels, then it will truncate on decode.